src/Stryker.Core/Stryker.Core/Stryker.Core.csproj

```diff
@@ -31,7 +31,11 @@
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+    <PackageReference Include="Microsoft.TestPlatform" Version="16.0.0-preview-20181205-02" />
+    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="16.0.0-dev" />
+    <PackageReference Include="Microsoft.TestPlatform.Portable" Version="16.0.0-preview-20181205-02" />
```

Comment:
Why the preview versions?

Reply:
Because there has not been a full release for a while, and the change we will need for vstest.console.dll integration will most likely be a preview version as well. This is also why vstest is not yet the default test runner.

Reply:
The current implementation only works on windows. Wouldn't it be best to use the last stable packages for now? When we add vstest for linux and mac we can update the package version.

Reply:
Agreed. Downgrading for now.